client/js/components/procedure/StatementSegmentsList/StatementMeta/StatementMetaMultiselect.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+  <div>
+    <dp-label
+      :text="label"
+      :for="name" />
+    <ul
+      v-if="editable"
+      :id="name"
+      class="o-list o-list--csv color--grey">
+      <template v-if="options.filter(opt => value.contains(opt))">
```

Comment:
Is method `contains` missing or do you mean `includes`?

Reply:
yes, this method is deprecated. As the logic has been changed slightly (the prop `value` is an `Array` now instead of a `String`). here is a new code https://github.com/demos-europe/demosplan-core/commit/f8798491b7ed3830e73cbb3fe0394c83b7fb74e9#diff-4729d18dcde3cfb52dbf7c49d6a03e9e0b6734cd10b775a102fda74c9f7bd8b3R106-R111